frontend/package.json

```diff
@@ -9,6 +9,10 @@
     "lint": "next lint"
   },
   "dependencies": {
+    "@emotion/react": "^11.11.1",
+    "@emotion/styled": "^11.11.0",
```

Comment:
Out of curiosity, what are these dependencies for?

Reply:
These are part of a CSS-in-Js lib that Joy-UI requires to work